include/pm/device.h

```diff
@@ -66,6 +66,13 @@ enum pm_device_state {
 enum pm_device_flag {
 	/** Indicate if the device is busy or not. */
 	PM_DEVICE_FLAG_BUSY,
+	/**
+	 * Indicates whether or not the device is capable of waking the system
+	 * up.
+	 **/
```

Comment:
Non-blocking comment style nit here.

Reply:
why ? this is consistent with comments in this file. The comment does not fit in one line.

Reply:
> why ? this is consistent with comments in this file. The comment does not fit in one line.


```
	 */
```

not

```
	 **/
```

Reply:
have changed it ;)